src/chunk.c

```diff
@@ -3207,32 +3218,21 @@ ts_chunk_do_drop_chunks(Hypertable *ht, int64 older_than, int64 newer_than, int3
 	{
 		int i;
 
-		/* Exclusively lock all chunks, and also refresh and invalidate the
+		/* Exclusively lock all chunks, and invalidate the
 		 * continuous aggregates in the regions covered by the chunks. Locking
 		 * prevents further modification of the dropped region during this
 		 * transaction, which allows moving the invalidation threshold without
 		 * having to worry about new invalidations while refreshing. */
```

Comment:
I think this comment needs to be updated. We are no longer moving the invalidation threshold here.